src/api/docs/doc.test.js

```diff
@@ -0,0 +1,31 @@
+const puppeteer = require('puppeteer');
+const { PORT } = require('../../config');
+const app = require('../../app').listen(PORT);
+
+let browser;
+let page;
+const BASE_URL = `http://localhost:${PORT}`;
+
+beforeAll(async () => {
+  browser = await puppeteer.launch();
+  page = await browser.newPage();
+});
+
+afterAll(async () => {
+  await browser.close();
+  app.close();
+});
+
+describe('GET /apiDocs/', () => {
+  it('Should display "Available versions" text on page', async () => {
+    await page.goto(`${BASE_URL}/apiDocs/`);
+    await expect(page).toMatch('Available versions');
```

Comment:
This await should be removed.

Reply:
Maybe, the browser closes before `page.goto` method has any chance to run. Tests are failing for that.